pkg/commands/oscommands/os_test.go

```diff
@@ -135,3 +137,43 @@ func TestOSCommandFileType(t *testing.T) {
 		_ = os.RemoveAll(s.path)
 	}
 }
+
+func TestOSCommandAppendLineToFile(t *testing.T) {
+	type scenario struct {
+		path  string
+		setup func(string)
+	}
+
+	scenarios := []scenario{
+		{
+			filepath.Join(os.TempDir(), "testFile"),
+			func(path string) {
+				if err := ioutil.WriteFile(path, []byte("hello"), 0o600); err != nil {
+					panic(err)
+				}
+			},
+		},
+		{
+			filepath.Join(os.TempDir(), "testFileWithNewline"),
```

Comment:
sorry for missing this the first time: could we also add a test for when the file is initially empty?